OpenRA.Mods.Common/AI/SupportPowerDecision.cs

```diff
@@ -94,7 +94,7 @@ public int GetAttractiveness(IEnumerable<Actor> actors, Player firedBy)
 		/// <summary>Makes up part of a decision, describing how to evaluate a target.</summary>
 		public class Consideration
 		{
-			public enum DecisionMetric { Health, Value, None }
+			public enum DecisionMetric { Health, Value, BaseBuilding, None }
```

Comment:
Like @pchote and @GraionDilach said above - add a new type (say "BaseBuilding" to the `Targetable` trait's `TargetTypes` property of the ConYard).
Then the AI will use something like
```
Consideration@1:
   Against: Enemy
   Types: BaseBuilding
   Attractiveness: 15
   TargetMetric: None
   CheckRadius: 5c0
```
without needing a new `TargetMetric`.

The attitude is completely unnecessary.